install_deps.sh

```diff
@@ -6,6 +6,8 @@ then
     # This older package libssl1.0-dev enables compiling Thrift 0.9.2
     # on Ubuntu 18.04.  Package libssl-dev exists, but Thrift 0.9.2
     # fails to compile when it is installed.
+    # TBD whether using this package makes a difference for Ubuntu
+    # 18.04 and Thrift 0.11.0.
```

Comment:
Let me know if anyone considers it important to determine whether we can use same libssl-dev package across Ubuntu 16.04 and 18.04 here.  The script as written works, but might also work with a common package for all supported Ubuntu versions.

Reply:
as long as it works :)